### PR TITLE
Improve contribution section

### PR DESCRIPTION
**Pull request by AKostiv8**

### CODE REVIEW

**Comment by AKostiv8** [comment]
https://akostiv8.github.io/gci16.fossasia.org/

**Comment by AKostiv8** [comment]
https://codein.withgoogle.com/dashboard/task-instances/5974474228760576/

**Comment by gaeun** [comment]
Can you use the Github API, and not a static array of the contributors?

File: javascripts/contributors.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "Hoangvanthien",
+      "Jig08",
+      "Hemantjadon",
+      "Gaeun",
+      "Ankitrgadiya",
+      "Princu7",
+      "Daminisatya",
+      "PolBaladas",
+      "Vickyjjj",
+      "BohJieQi",
```
Comment (abishekvashok) [inline javascripts/contributors.js]: Dont use static array use the github api because in it contributors are added automatically...

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please use github api, we already have a contributors section please try to enhance it and make use of github api.

**Comment by AKostiv8** [comment]
I don't know how to use Github API

**Comment by AKostiv8** [comment]
@niccokunzmann

**Comment by niccokunzmann** [comment]
@AKostiv8 How would you like to go on? Do you have plans? Would you like suggestions?

**Comment by AKostiv8** [comment]
hurry up!!! I have done it))))))

**Comment by gaeun** [comment]
@AKostiv8 I visited your live link, and it seems like the contributors aren't showing up... But I think a bit of debugging would make it work! Great job!

File: stylesheets/contributors.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  line-height: 40px;
+}
+.visible:after, .visible:before {
+  position: absolute;
+  top: 0;
+  color: #16a085;
+  font-size: 42px;
+  -webkit-animation-name: opacity;
+  -webkit-animation-duration: 2s;
+  -webkit-animation-iteration-count: infinite;
```
Comment (robbyoconnor) [inline stylesheets/contributors.css:42]: Same here

File: stylesheets/contributors.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+ul {
+  margin-top: 0;
+  margin-left: 90px;
+  padding-left: 410px;
+  text-align: left;
+  list-style: none;
+  -webkit-animation-name: change;
+  -webkit-animation-duration: 6s;
+  -webkit-animation-iteration-count: infinite;
```
Comment (robbyoconnor) [inline stylesheets/contributors.css:71]: Same here.

File: stylesheets/contributors.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
+.content {
+  position: absolute;
+  margin-top: 7%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
```
Comment (robbyoconnor) [inline stylesheets/contributors.css:5]: We need to support other browsers too...not just webkit browsers...if you're gonna use vendor-prefixes handle them all.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
If we include vendor prefixes in our CSS, we should include them for other browsers as well..

**Comment by robbyoconnor** [comment]
@AKostiv8, are you going to fix this?

**Comment by abishekvashok** [comment]
@AKostiv8 nice job, keep up the good work :) now improve it all together, if you want help ping on slack.

**Comment by AKostiv8** [comment]
@Abhi2424shek @robbyoconnor  I have fixed your requested

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
@AKostiv8 nice work, bro can you squash your commits see the contributing.md file for info and tutorials on squashing if they don't help ping me on slack I would help you out for sure so happy coding :)

After you squash and @robbyoconnor reviews we will merge this!

**Comment by AKostiv8** [comment]
@Abhi2424shek  thanks for supporting

**Comment by AKostiv8** [comment]
@Abhi2424shek  I have squashed it

File: stylesheets/contributors.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  -mos-transform: translate(-50%, -50%);
```
Comment (robbyoconnor) [inline stylesheets/contributors.css]: Don't you `moz-transform`

File: stylesheets/contributors.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  position: absolute;
+  top: 0;
+  color: #16a085;
+  font-size: 42px;
+  -webkit-animation-name: opacity;
+  -webkit-animation-duration: 2s;
+  -webkit-animation-iteration-count: infinite;
+  -mos-animation-name: opacity;
+  -mos-animation-duration: 2s;
+  -mos-animation-iteration-count: infinite;
```
Comment (robbyoconnor) [inline stylesheets/contributors.css]: The prefix for mozilla is `moz`

File: stylesheets/contributors.css [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  margin-left: 90px;
+  padding-left: 410px;
+  text-align: left;
+  list-style: none;
+  -webkit-animation-name: change;
+  -webkit-animation-duration: 6s;
+  -webkit-animation-iteration-count: infinite;
+  -mos-animation-name: change;
+  -mos-animation-duration: 6s;
+  -mos-animation-iteration-count: infinite;
```
Comment (robbyoconnor) [inline stylesheets/contributors.css]: `moz`

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
The mozilla vendor prefix is incorrect.

**Comment by robbyoconnor** [comment]
Here is a [guide on vendor prefixes](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix).

**Comment by robbyoconnor** [comment]
@Abhi2424shek DO NOT MERGE! The vendor prefixes for firefox are incorrect.

**Comment by robbyoconnor** [comment]
Also there's a quirk with the thing thanking contributors: 
image

See how the bottom portion of the `y` is not visible?

**Comment by robbyoconnor** [comment]
@AKostiv8 you still have problems you have to fix. Learn about the vendor prefixes please :wink:

**Comment by abishekvashok** [comment]
@AKostiv8 status

**Comment by abishekvashok** [comment]
> @Abhi2424shek DO NOT MERGE! The vendor prefixes for firefox are incorrect.

@robbyoconnor I would only merge after proper testing

**Comment by robbyoconnor** [comment]
Everything is perfect but the spacing on the ticker thingy is awkward

image

Either center it in that whitespace or clean it up. As it stands, it's not good.

File: stylesheets/contributors.css [hunk 8]
```diff
@@ -18,7 +18,7 @@
 .visible {
   font-weight: 600;
   overflow: hidden;
-  height: 40px;
+  height: 50px;
```
Comment (robbyoconnor) [inline stylesheets/contributors.css:21]: Needs to be reduced. Play around with it a bit...

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please fix as suggested

**Comment by AKostiv8** [comment]
@robbyoconnor  I have fixed requested

**Comment by AKostiv8** [comment]
@robbyoconnor  @Abhi2424shek  can you approved

**Comment by AKostiv8** [comment]
?

**Comment by abishekvashok** [comment]
@AKostiv8 @robbyoconnor looks good for me:
image

@robbyoconnor waiting for you sir

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
@AKostiv8 can you squash the commits.

**Review by robbyoconnor — APPROVED** [review]

**Comment by robbyoconnor** [comment]
Merging.

**Comment by ghost** [comment]
thanks

**Comment by ghost** [comment]
@robbyoconnor  can you approve?
https://codein.withgoogle.com/dashboard/task-instances/5974474228760576/

**Pull request merged**